app/scripts/modules/core/src/pipeline/config/stages/stage.module.js

```diff
@@ -178,6 +178,9 @@ module.exports = angular
           $scope.description = config.description;
           $scope.extendedDescription = config.extendedDescription;
           $scope.label = config.label;
+          if (config.configuration && config.configuration.failOnFailedExpressions) {
```

Comment:
yo dawg, maybe we can call this `config.defaults` or something else?

Reply:
I actually started with `config.defaults`. :)

But `config.configuration` is already a thing, and it felt ok to chuck it in there instead of creating something new.

Reply:
wow online @christopherthielen uses much more slang than IRL @christopherthielen

Reply:
Went back to `config.defaults` since that can also be used for other things.